### PR TITLE
Reinstates movie cards.

### PR DESCRIPTION
**Pull request by chuckharmston**
screen shot 2016-09-26 at 5 27 11 pm

This reinstates the movie cards commit ~~without A/B testing~~, with two required fixes and A/B testing.

r? @6a68

### CODE REVIEW

**Comment by jaredhirsch** [comment]
@chuckharmston Looking at this PR, it appears that the commit `957c823f` adds 442 lines and removes 216. This makes it a nearly exact, but not exact, copy-paste of the original commit from pull request #261, which added 440 lines, and removed 211. Could you either revert the original commit and add a separate commit to change those 5-10 lines, or document in this PR which lines were changed?

I don't understand the variant code at a glance, I'll test it out and add review comments tomorrow morning.

**Comment by chuckharmston** [comment]
As I noted on IRC, 957c823f is a revert commit with a few conflicts fixed in `lib/universal-search.js`.

**Comment by jaredhirsch** [comment]
@chuckharmston Cool. And what was wrong with the other A/B PR? Was it not working?

**Comment by chuckharmston** [comment]
@6a68 It wasn't, [as I noted here](https://github.com/mozilla/universal-search/pull/278#issuecomment-249736167). At the very least the [order of the arguments here](https://github.com/mozilla/universal-search/pull/278/files#diff-6564594474c8f8abbaea8eb3a131b5c5R85) is wrong, but correcting that still didn't fix it. I'm not sure how you ever had it working at all.

Since you weren't available for questions, I started mostly from scratch, using code I wrote for [one of the example add-ons](https://github.com/mozilla/testpilot/tree/master/docs/examples/addon-restartless) as a base.

**Comment by jaredhirsch** [comment]
Cool. Too bad the old PR wasn't working. Looking now

File: lib/ui/recommendation.js [hunk 1]
```diff
@@ -159,48 +164,83 @@ Recommendation.prototype = {
...
+    getResultType contains the rules used to decide if a given data packet
+    should be shown as a TLD, movie, or Wikipedia recommendation type.
+
+    The movie card is only shown to a fraction of users; the Test Pilot add-on
+    selects which users are in which group.
+    */
+    let type;
+    if ('tld' in data.enhancements) {
+      type = 'tld';
+    } else if (this.win.universalSearch.variants.useMovieCard &&
```
Comment (jaredhirsch) [inline lib/ui/recommendation.js]: Looks like this should be `variants.movieCard`, not `useMovieCard`

**Comment by jaredhirsch** [comment]
@chuckharmston Nice work! I do see movie cards when I correct that `variants.useMovieCard` line, and manually set `universalSearch.variants.movieCard = true`.

Edited: after pulling in that latest recommendation server commit, I'm now seeing images. They are arriving rather late, but hopefully the image proxy will help when this actually hits production.

Feel free to land this one the variants check is fixed.

search-movie-cards-2

**Comment by jaredhirsch** [comment]
@chuckharmston Since you're on a flight, and we want QA to look at this as soon as possible, I went ahead and changed that one line in ui/recommendation.js from `variants.useMovieCard` to `variants.movieCard`, and force-pushed to your branch. Merging.

**Pull request merged**